src/core/renderers/webgl/filters/SpriteMaskFilter.js

```diff
@@ -13,6 +12,7 @@ var AbstractFilter = require('./AbstractFilter'),
 function SpriteMaskFilter(sprite)
 {
     var maskMatrix = new math.Matrix();
+    var fs = require('fs');
```

Comment:
Try the current dev. It won't build. It does build when moving it into the constructor for some reason.